native/src/components/RemoteContent.tsx

```diff
@@ -167,7 +167,6 @@ const RemoteContent = ({
       }}
       originWhitelist={['*']} // Needed by iOS to load the initial html
       javaScriptEnabled
-      dataDetectorTypes='none'
```

Comment:
🤔 This causes a change in behavior as per default phone numbers are detected, was this a change on purpose? https://github.com/react-native-webview/react-native-webview/blob/master/docs/Reference.md#datadetectortypes

Reply:
Yes, it was. Keeping it like this led to the app crashing in iOS. I'll see if I find a different way to disable all automatic data detection